docs/interfaces/desktopagent.md

```diff
@@ -89,14 +107,35 @@ ___
 
 ▸ **open**(name: *`String`*, context?: *[Context](../#context)*): `Promise`<`void`>
 
-*Defined in [interface.ts:83](/src/interface.ts#L83)*
+*Defined in [interface.ts:104](/src/interface.ts#L104)*
 
 Launches/links to an app by name.
 
 If a Context object is passed in, this object will be provided to the opened application via a contextListener. The Context argument is functionally equivalent to opening the target app with no context and broadcasting the context directly to it.
 
 If opening errors, it returns an `Error` with a string from the `OpenError` enumeration.
 
+```javascript
+//no context
+    agent.open('myApp');
+    //with context
```

Comment:
👍 Thanks for showing both versions, that's good to have. The same comment as above goes for including the shape of the context data here.